app/views/hyrax/base/_show_actions.html.erb

```diff
@@ -13,6 +13,12 @@
           <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
             Attach Child <span class="caret"></span>
           </button>
+
+          <input type="checkbox"  style="display:none" name="batch_document_ids[]" id="batch_document_<%= presenter.id %>" value="<%= presenter.id %>" class="batch_document_selector" checked="checked" />
+          <%= button_tag t('hyrax.dashboard.my.action.add_to_collection'),
+              class: 'btn btn-default submits-batches submits-batches-add',
+              data: { toggle: "modal", target: "#collection-list-container", turbolinks: false } %>
```

Comment:
Try taking off `turbolinks: false`.  There was a previous PR that should make that unnecessary.  So we don't want to add another use of it unless it is required.

Reply:
will try!